kr/kr_unix.go

```diff
@@ -20,11 +20,14 @@ func getPrefix() string {
 	return prefix
 }
 
-func restartCommand(c *cli.Context) (err error) {
+func restartCommandOptions(c *cli.Context, showPrints bool) (err error) {
 	kr.Analytics{}.PostEventUsingPersistedTrackingID("kr", "restart", nil, nil)
 	exec.Command("pkill", "krd").Run()
 	startKrd()
-	PrintErr(os.Stderr, "Restarted Kryptonite daemon.")
+
+	if showPrints {
```

Comment:
same here regarding `showPrints` renaming and analytics

Reply:
👍